### PR TITLE
Use system breakepad on unix

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -16,6 +16,14 @@ endif()
 
 option(ENABLE_CRASH_REPORTER "Enable libcrashreporter-qt GUI component" ON)
 
+if (NOT MSVC)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(CLOUDPROVIDERS breakpad IMPORTED_TARGET)
```
Comment (fmoc) [inline CMakeLists.txt:21]: I think this line should be `BREAKPAD`.

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 project(libcrashreporter-qt)
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.12)
```
Comment (fmoc) [inline CMakeLists.txt:2]: 3.6 would suffice for `IMPORTED_TARGET`. Any other reasons why you bumped it to 3.12?

File: 3rdparty/breakpad/CMakeLists.txt [hunk 3]
```diff
@@ -72,14 +71,20 @@ if(WIN32)
...
+endif()
 
 if(NOT MSVC)
     target_link_libraries(crashreporter-breakpad pthread)
 endif()
 
-if(WIN32 AND NOT MSVC)
-    target_link_libraries(crashreporter-breakpad "mingwex")
-endif()
+add_library(PkgConfig::BREAKPAD ALIAS crashreporter-breakpad)
```
Comment (fmoc) [inline 3rdparty/breakpad/CMakeLists.txt:90]: I think you should move this line and the definition of the custom target into the `if(WIN32)`. There is no option you can set to link to the custom `breakpad` on non-Windows anyway.

File: CMakeLists.txt (hunk 1 above)
Comment (TheOneRing) [reply CMakeLists.txt:21]: 🙈

**Comment by TheOneRing** [comment]
Updated

**Comment by TheOneRing** [comment]
See https://github.com/dschmidt/libcrashreporter-qt/pull/35

**Pull request closed without merging**